.github/ISSUE_TEMPLATE/build_error.md

```diff
@@ -0,0 +1,83 @@
+---
+name: Build error 
+about: Some package in Spack didn't build correctly  
+
+---
+
+*Thanks for taking the time to report this build failure. To proceed with the
+report please:*
+1. Title the issue "Installation issue: <name-of-the-package>".
+1. Provide the information required below.
+1. Clean the issue from the template instructions below when finished.
```

Comment:
If this is saying "there are instructions below on how to clean the issue template" I don't see them. If this is just saying "Remove the template instructions before posting the issue" then I think that may be a clearer way to say it.